proposals/expand-ref.md

```diff
@@ -104,11 +107,19 @@ void M(
 The [method arguments must match](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-11.0/low-level-struct-improvements.md#rules-method-arguments-must-match) rules will be updated to take `ref scoped` into account. Values passed to such parameters do not need to be considered when calculating the return scopes.
 
 Detailed notes:
+
 - A `ref scoped` parameter is implicitly `scoped ref`
 - An `out scoped` parameter declaration is an error
 
+### ref scoped locals
+
+The language will allow for locals to be declared as `ref scoped`. This will serve to have the lifetime of the value be _current method_. This lifetime is exactly _current method_ unlike paramteers which are just known to be wider than _current method_.
```

Comment:
```suggestion
The language will allow for locals to be declared as `ref scoped`. This will serve to have the lifetime of the value be _current method_. This lifetime is exactly _current method_ unlike parameters which are just known to be wider than _current method_.
```